server/src/form_processor/queries.go

```diff
@@ -189,7 +191,7 @@ select
         '',
         '', 
         NULL,
-        '47',
+        '` + model.SFBayChapterIdStr + `',
```

Comment:
❓  does this not work for this one?

```suggestion
        model.SFBayChapterIdStr,
```

Reply:
this is a list in SQL in the middle of a huge `backtick string`, not a Golang function call or other list in Go :)